<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewRegisterCourses</class>
 <widget class="QDialog" name="viewRegisterCourses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="StudName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="StudLvl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Level</string>
   </property>
  </widget>
  <widget class="QListWidget" name="coursesList">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
